src/main/java/org/reaktivity/nukleus/amqp/internal/stream/AmqpServerFactory.java

```diff
@@ -176,6 +176,8 @@
     private static final long DEFAULT_IDLE_TIMEOUT = 0;
     private static final int READ_IDLE_SIGNAL_ID = 0;
     private static final int WRITE_IDLE_SIGNAL_ID = 1;
+    private static final int CLOSE_SIGNAL_ID = 2;
+    private static final int CLOSE_TIMEOUT = 500;
```

Comment:
Please shift `CLOSE_TIMEOUT` to `AmqpConfiguration.CLOSE_EXCHANGE_TIMEOUT`, defaulted to `10000ms`, then override in IT to keep build time down while still validating state machine behavior.